packages/@aws-cdk/toolkit/lib/api/io/private/codes.ts

```diff
@@ -1,20 +1,27 @@
 import { IoMessageCode } from '../io-message';
 
 export const CODES = {
-  // Toolkit Info codes
-  CDK_TOOLKIT_I0001: 'Display stack data',
-  CDK_TOOLKIT_I0002: 'Successfully deployed stacks',
-  CDK_TOOLKIT_I3001: 'Informs about any log groups that are traced as part of the deployment',
-  CDK_TOOLKIT_I5001: 'Display synthesis times',
-  CDK_TOOLKIT_I5050: 'Confirm rollback',
+  // Synth
+  CDK_TOOLKIT_I1000: 'Provides synthesis times',
+  CDK_TOOLKIT_I1901: 'Provides stack data',
+  CDK_TOOLKIT_I1902: 'Successfully deployed stacks',
```

Comment:
Is there a way to codify these rules, i.e. that Synth messages must start with `1XXX`, better than the honor system we're employing right now?

Reply:
Fair. I thought about it some and I couldn't think of anything reasonable without supplying additional information into the record set similar to how lambda runtimes work. i.e. `static readonly CDK_TOOLKIT_I1000 = new Code('synth', 'Provides synthesis times')`

But i think its not worth investigating further so i'm approving.